tests/lib/processors/gcp_logging_timesketch.py

```diff
@@ -102,7 +96,9 @@ def testGCEFirewallLog(self):
         'message':
             'User heinz-57@ketchup-research.iam.gserviceaccount.com '
             'performed v1.compute.firewalls.insert '
-            'on projects/ketchup-research/global/firewalls/deny-tomchop-access'
+            'on projects/ketchup-research/global/firewalls/deny-tomchop-access',
+        'severity':
+            'NOTICE'
```

Comment:
```suggestion
        'severity': 'NOTICE'
```

Same in other places